src/installCommand/index.js

```diff
@@ -108,9 +109,46 @@ const installationSpec = {
         await applyPatch(packageJson, target);
       }
     },
+
+    installDidComplete: async (installedPackages) => {
+      if (!installedPackages) {
+        return;
+      }
+      await linkConditionalDependencies(installedPackages);
```

Comment:
Should this be done via some kind of `installWillLink` API hook? Is it fragile or discouraged to "link" in this way?